make-it/EmMakefile

```diff
@@ -50,8 +64,15 @@ help:
 #  Vireo has grown as well, so emscripten is getting better if the over all size has shrunk by about 20%
 
 EMCC= emcc
-EM_DBGFLAG = -g4
-EM_OPTFLAG = -Os # -Os -O3
+EM_OPTFLAG = -O3 #release config
+
+ifeq ($(BUILD), debug)
+   EM_OPTFLAG = -g4
```

Comment:
Consider enabling flags already included in Custom.mak.

Reply:
Good catch